<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Secure Chat - Login</string>
  </property>
  <property name="windowOpacity">
   <double>0.990000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(54, 54, 54)</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:26pt; color:rgb(141, 141, 211)</string>
   </property>
   <property name="text">
    <string>Secure Chat</string>
   </property>
  </widget>
  <widget class="QLabel" name="username_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14pt;color:rgb(202, 100, 151)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>139</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(232, 232, 232);font-size:14pt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>219</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(232, 232, 232);font-size:14pt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14pt;color:rgb(202, 100, 151)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="username_label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 10pt;color:rgb(202, 100, 151)</string>
   </property>
   <property name="text">
    <string>Don't have an account? Create one here!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createAccountButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(167, 167, 167)</string>
   </property>
   <property name="text">
    <string>Create Account</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(167, 167, 167)</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
